_layouts/es/default.html

```diff
@@ -7,7 +7,7 @@
     <link href="/atom.xml" rel="alternate" title="Alaveteli" type="application/atom+xml">
     <link rel="icon" type="image/png" href="/assets/img/favicon.png" />
     <meta name="viewport" content="initial-scale=1">
-    <link href='http://fonts.googleapis.com/css?family=Source+Sans+Pro:400,600,700,900,400italic&subset=latin,latin-ext' rel='stylesheet' type='text/css'>
+    <link href='https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,600,700,900,400italic&subset=latin,latin-ext' rel='stylesheet' type='text/css'>
```

Comment:
I _think_ we should be able to do `<link href='//fonts…` as we've done for `<script src="//ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>` a few lines below?

Reply:
Actually its now considered bad practice to do this https://webhint.io/docs/user-guide/hints/hint-no-protocol-relative-urls/#why-is-this-important, so I think this is fine.